src/scripts/release.zig

```diff
@@ -140,19 +142,172 @@ fn build(shell: *Shell, languages: LanguageSet, info: VersionInfo) !void {
     }
 }
 
+/// Does the same thing as llvm-lipo (builds a universal binary) but allows building binaries
+/// that have deprecated architectures. This is used by multiversioning on macOS, where these
+/// deprecated architectures hold the metadata and version packs.
+/// It's much easier to embed and read them here, then to do it in the inner MachO binary, like
+/// we do with ELF or PE.
+fn build_macos_universal_binary(
```

Comment:
Nit, but move `build_past_version_pack`/`build_macos_universal_binary` down below `build_tigerbeetle` since this file is mostly organized "top-down".